src/integrationTest/java/uk/gov/hmcts/darts/common/controller/GetWelcomeTest.java

```diff
@@ -24,4 +27,115 @@ void welcomeRootEndpoint() throws Exception {
 
         assertThat(response.getResponse().getContentAsString()).startsWith("Welcome");
     }
+
+    @Test
+    void dailyListAddDailyListEndpoint() throws Exception {
```

Comment:
I think this should probably be moved into a different file? Like `DailyListTest.java` or something?

Reply:
done